common-queries/src/main/scala/queries/schemas/odb/ObsQueries.scala

```diff
@@ -125,10 +130,13 @@ object ObsQueries:
             constraints = obs.constraintSet,
             targets = obs.targetEnvironment,
             posAngle = obs.posAngleConstraint,
+            offsets =
+              NonEmptyList.fromList(data.sequence.foldMap(_.executionConfig.allOffsets).distinct),
```

Comment:
Maybe we want a `NonEmptySet` ?

Reply:
Could be, order is important, would Set preserve it?

Reply:
No, I thought order was not important.